CleverPush/Source/CPWKWebView.m

```diff
@@ -1,6 +1,7 @@
 #import "CPWKWebView.h"
-
-@interface CPWKWebView () <WKNavigationDelegate, WKUIDelegate>
+#import "CPStoriesController.h"
+#import "CPLog.h"
+@interface CPWKWebView () <WKNavigationDelegate, WKUIDelegate, storyViewOpenedListener>
 @end
 
 @implementation CPWKWebView
```

Comment:
Add `urlOpenedListener` as a variable to this class. Customer can then set this listener (like NotificationOpenedListener in CleverPushInstance.m)